elftools/elf/descriptions.py

```diff
@@ -237,30 +237,30 @@ def describe_note(x, machine):
 
 
 def describe_attr_tag_arm(tag, val, extra):
+    s = _DESCR_ATTR_TAG_ARM.get(tag, '"%s"' % tag)
```

Comment:
Why do we have this? Unknown tags not defined in `_DESCR_ATTR_TAG_ARM`?

Also, what's the point of the string interpolation? Isn't `tag` already a string?